ziti/options.go

```diff
@@ -61,13 +61,42 @@ func (d DialOptions) GetConnectTimeout() time.Duration {
 }
 
 type ListenOptions struct {
-	Cost                         uint16
-	Precedence                   Precedence
-	ConnectTimeout               time.Duration
-	MaxConnections               int
-	Identity                     string
-	BindUsingEdgeIdentity        bool
-	ManualStart                  bool
+	// Initial static cost assigned to terminators for this service
+	Cost uint16
+
+	// Initial precedence assigned to terminators for this service
+	Precedence Precedence
+
+	// When using WaitForNEstablishedListeners, how long to wait before giving
+	// if N listeners can't be established
+	ConnectTimeout time.Duration
+
+	// Maximum number of terminators to establish. If a value less than 1 is provided,
+	// will default to 1. At most one terminator will be established per available
+	// edge router. If both MaxConnections and MaxTerminators have non-zer values,
+	// the value from MaxTerminators will be used
+	//
+	// Deprecated: used MaxTerminators instead.
+	MaxConnections int
+
+	// Maximum number of terminators to establish. If a value less than 1 is provided,
+	// will default to 1. At most one terminator will be established per available
+	// edge router. If both MaxConnections and MaxTerminators have non-zer values,
+	// the value from MaxTerminators will be used
+	MaxTerminators int
```

Comment:
Assuming you're leaving these both in here for backwards compatibility?

Reply:
Yes, I figured we'd remove it in a few versions.